docs/en/wallet-instance-attestation.rst

```diff
@@ -2,56 +2,676 @@
+
+Verifiable through the public key of the wallet provider itself.
+
+Endpoints
+~~~~~~~~~
+The backend wallet that issues the wallet instance attestations must
+make available a series of APIs in REST format that follow the OpenId
+Federation standard.
 
+Metadata
+^^^^^^^^
+A **GET /.well-known/openid-federation endpoint** for retrieving the Wallet
+Provider Entity Configuration
 
-Attributes
-----------
+Wallet Instance Attestation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-[Table with parameters/attributes]
-    
-.. list-table:: 
-   :widths: 20 60
-   :header-rows: 1
+A second **POST /token** endpoint that takes two parameters as input:
 
-   * - **Claim**
-     - **Description**
-   * - key
-     - value
+``grant_type`` which in our case is a string:
+``urn:ietf:params:oauth:grant-type:wallet-instance-attestation``
 
-          
-Implementation considerations
------------------------------
+``assertion``` which contains the JWT of the Wallet Instance Attestation
+Request
 
-TODO
+The response will then contain the Wallet Instance Attestation
 
+Open API specifications
+^^^^^^^^^^^^^^^^^^^^^^^
 
-Libraries and code snippets
---------------------------- 
+.. code-block:: yaml
 
-TODO
+  openapi: 3.0.0
```

Comment:
why is this required, since we have the wallet provider metadata ?

Reply:
wallet provider metadata only provides the list of endpoints but does not define how to integrate (e.g. POST/GET etc). Maybe in these specifications we can also eliminate this part

Reply:
yes, I agree
we can live without it